bomba/roles/mosquitto-server/files/etc/mosquitto/conf.d/bitraf.acl

```diff
@@ -130,3 +133,8 @@ topic write bitraf/nexa/1/cmd/#
 
 user bitraf-hotbot
 topic write bitraf/octoprint/hotbot/#
+
+# machine access
+user bitraf-p2k16-web
```

Comment:
Må passe på å skape denne nye brukeren. Hvis ikke tror jeg Mosquitto vil nekte å starte

Reply:
`bitraf-p2k16-web` fantes allerede i fila, for `/bitraf/door/+/open`

Reply:
Ok!